Decisions.Equifax/PrequalificationOfOne/Response/PreQualificationOfOneResponse.cs

```diff
@@ -0,0 +1,611 @@
+            public ScoreNumberOrMarketMaxIndustryCode ScoreNumberOrMarketMaxIndustryCode { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class ScoreNumberOrMarketMaxIndustryCode
+        {
+            [JsonProperty("code")] public string Code { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class RiskBasedPricingOrModel
+        {
+            [JsonProperty("percentage")] public string Percentage { get; set; }
+
+            [JsonProperty("lowRange")] public long LowRange { get; set; }
+
+            [JsonProperty("highRange")] public long HighRange { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class NameMatchFlags
+        {
+            [JsonProperty("firstNameMatchFlag")] public string FirstNameMatchFlag { get; set; }
+
+            [JsonProperty("lastNameMatchFlag")] public string LastNameMatchFlag { get; set; }
+
+            [JsonProperty("middleNameMatchFlag")] public string MiddleNameMatchFlag { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class OfacAlert
+        {
+            [JsonProperty("revisedLegalVerbiageIndicator")]
+            public long RevisedLegalVerbiageIndicator { get; set; }
+
+            [JsonProperty("memberFirmCode")] public string MemberFirmCode { get; set; }
+
+            [JsonProperty("cdcTransactionDate")] public string CdcTransactionDate { get; set; }
+
+            [JsonProperty("cdcTransactionTime")] public long CdcTransactionTime { get; set; }
+
+            [JsonProperty("transactionType")] public string TransactionType { get; set; }
+
+            [JsonProperty("cdcResponseCode")] public string CdcResponseCode { get; set; }
+
+            [JsonProperty("legalVerbiage")] public string LegalVerbiage { get; set; }
+
+            [JsonProperty("dataSegmentRegulated")] public string DataSegmentRegulated { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public class OnlineGeoCode
+        {
+            [JsonProperty("streetNumber")] public long StreetNumber { get; set; }
+
+            [JsonProperty("streetName")] public string StreetName { get; set; }
+
+            [JsonProperty("streetTypeOrDirection")]
+            public string StreetTypeOrDirection { get; set; }
+
+            [JsonProperty("city")] public string City { get; set; }
+
+            [JsonProperty("stateAbbreviation")] public string StateAbbreviation { get; set; }
+
+            [JsonProperty("zipCode")] public string ZipCode { get; set; }
+
+            [JsonProperty("typeOfAddress")]
+            [JsonConverter(typeof(FraudVictimIndicatorDataConverter))]
+            public FraudVictimIndicator TypeOfAddress { get; set; }
+
+            [JsonProperty("returnCode1")]
+            [JsonConverter(typeof(FraudVictimIndicatorDataConverter))]
+            public FraudVictimIndicator ReturnCode1 { get; set; }
```

Comment:
ReturnCode may be the correct name for it.

Reply:
according to the sample data it is actually returnCode1 thru 4